backend/app/services/execution/polling_dispatcher.py

```
@@ -0,0 +1,346 @@
+                        subtask_id=subtask_id,
+                        message_id=message_id,
+                    )
+                )
+                return
+
+            # Check status
+            try:
+                status_result = await gemini_client.get_interaction_status(job_id)
+            except GeminiInteractionError:
+                logger.warning(
+                    f"[PollingDispatcher] Status check failed, retrying..."
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)